proof/crefine/RISCV64/Retype_C.thy

```diff
@@ -7258,8 +7312,7 @@ lemma range_cover_n_le':
   done
 
 lemma createNewObjects_ccorres:
-notes blah[simp del] =  atLeastAtMost_iff atLeastatMost_subset_iff atLeastLessThan_iff
-      Int_atLeastAtMost atLeastatMost_empty_iff split_paired_Ex
+notes blah[simp del] = atLeastAtMost_simps
```

Comment:
```suggestion
notes [simp del] = atLeastAtMost_simps
```

Reply:
The proof for this result is currently sorried, so I'm not sure whether the `blah` name can be removed: it does currently occur in the (commented-out part of the) proof. I'll put a `FIXME RT` asking whether we can remove the `blah` name when the proof is done